packages/app/src/game/module/board/boardContainer.tsx

```diff
@@ -97,7 +99,7 @@ const BoardContainer: React.FunctionComponent<{ showNowPlaying?: boolean }> = ({
                         )}
                     />
                 </div>
-
+                {inReadyPhase && <ReadyOverlay/>}
```

Comment:
Put this logic into the overlay itself, it stops the `BoardContainer` from being concerned here when it doesn't need to be

Reply:
* removed inReadyPhase variable from BoardContainer and repositioned logic in component itself